ingress/controllers/nginx/nginx.tmpl

```diff
@@ -265,6 +265,12 @@ http {
         {{ end }}
 
         {{ if eq $server.Name "_" }}
+        # health checks in cloud providers require the use of port 80
+        location /ingress-controller-healthz {
```

Comment:
any reason it shouldn't be a flag? if i have an ingress controller behind an ingress controller how would i modify it :)